.github/workflows/deploy-docker.yml

```diff
@@ -0,0 +1,36 @@
+name:                           Docker Image Release
+
+on:
+  push:
+    branches:
+      - stable
+    tags:
+      - v2*
```

Comment:
Is the plan is to release v2.8.0 and then switch to v3.0? Or is there a plan at all? :)

Reply:
yes, for now internally v2.7.3 to test the flow. Then v3 will be done and communicated externally, that will need a change in this param here and also in the other github action scripts.

How the process will look like we will not now exactly until it's done